### PR TITLE
v1.18.10 support

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: minecraft/protocol/chunk_pos.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// ChunkPos is the position of a chunk. It is composed of two integers, and is typically written as either
+// two varint32s.
+type ChunkPos [2]int32
+
+// X returns the X coordinate of the chunk position. It is equivalent to ChunkPos[0].
+func (pos ChunkPos) X() int32 {
+	return pos[0]
+}
+
+// Z returns the Z coordinate of the chunk position. It is equivalent to ChunkPos[2].
```
Comment (DaPigGuy) [inline minecraft/protocol/chunk_pos.go:12]:
```suggestion
// Z returns the Z coordinate of the chunk position. It is equivalent to ChunkPos[1].
```

File: minecraft/protocol/chunk_pos.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package protocol
+
+// ChunkPos is the position of a chunk. It is composed of two integers, and is typically written as either
+// two varint32s.
```
Comment (TwistedAsylumMC) [inline minecraft/protocol/chunk_pos.go:4]: Was this copied from somewhere that let the position be written in two ways? You should probably remove the "either"

File: minecraft/protocol/packet/add_volume_entity.go [hunk 3]
```diff
@@ -5,12 +5,18 @@ import (
...
 	EntityRuntimeID uint64
-	// VolumeEntityData ...
-	VolumeEntityData map[string]interface{}
+	// EntityMetadata is a map of entity metadata, which includes flags and data properties that alter in
+	// particular the way the entity looks.
+	EntityMetadata map[string]interface{}
+	// EncodingIdentifier ...
+	EncodingIdentifier string
+	// InstanceIdentifier ...
+	InstanceIdentifier string
```
Comment (TwistedAsylumMC) [inline minecraft/protocol/packet/add_volume_entity.go:19]: Is there no documentation for this stuff? I believe this volume stuff is in behaviour packs so we could probably RE it

**Review by T14Raptor — CHANGES_REQUESTED** [review]

File: minecraft/protocol/packet/add_volume_entity.go (hunk 3 above)
Comment (JustTalDevelops) [reply minecraft/protocol/packet/add_volume_entity.go:19]: I'm not exactly sure but I'll do some digging and update you if I find anything.

Comment (Sandertv) [reply minecraft/protocol/packet/add_volume_entity.go:19]: This PR should be good to go once this is resolved.

Comment (JustTalDevelops) [reply minecraft/protocol/packet/add_volume_entity.go]: Resolved.

**Pull request merged**